Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build Stage
+FROM golang:1.12 as build
```

Comment:
just use `FROM golang:1 ...`
as it'll pick up the latest stable golang 1.x and you won't be stuck in the past.

Reply:
"latest" shows intent a bit better and is functionally equivalent

Reply:
Once it starts pointing to go2 it might be an issue.

Reply:
I'll leave at go1 and update the comment to better reflect.